packages/world/ts/config/v2/output.ts

```diff
@@ -8,4 +51,13 @@ export type Config = StoreConfig & {
       };
     };
   };
+  readonly systems: SystemsConfig;
+  /** Systems to exclude from automatic deployment */
```

Comment:
system names?

Reply:
are you referring to the comment for `excludeSystems`?

Reply:
yeah, I think this is system contract names to exclude?